lib/config/presets/internal/replacements.ts

```diff
@@ -38,6 +39,54 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'emotion-10-to-11': {
+    description:
+      'various unscoped emotion v10 packages became renamed & scoped in v11',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@emotion/core', 'emotion-theming'],
+        replacementName: '@emotion/react',
+        replacementVersion: '11.0.0',
```

Comment:
Does `replacementVersion` get treated as a range by default, or does it only support replacing to an explicit version? It's not clear to me from the docs or the single unit test related to replacements.

* [handles replacements](https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/process/lookup/index.spec.ts#L1598)
* [handles replacements snap](https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/process/lookup/__snapshots__/index.spec.ts.snap#L196)

Reply:
It's exact version.

Reply:
Ok, thanks.